### ISSUE TITLE
Should be able to handle LF terminated Multipart Boundries

### ISSUE THREAD

**Issue by e-moran** [cetra3/mpart-async#24]
mpart-async currently returns an `InvalidBoundry` error if a multipart boundary uses the LF line terminator. While the HTTP Spec does call for CRLF terminated Multipart Boundries,[ it also state that applications should be tolerant where possible](https://www.rfc-editor.org/rfc/rfc2616#section-19.3). I am currently writing an application that I would like to be tolerant in this regard, but have to work around this in mpart-async by replacing the line endings of the data that I provide to it.

I'm happy to submit a PR to make this change if you agree that it is something that you'd like to support.

**Comment by cetra3** [comment]
Yes that is definitely something that we should support going forward.  Happy for you to write a PR!

I don't know if I would want mixed `\n` and `\r\n` within the same request, I.e, it should use either one or the other.
